CMakePresets.json

```diff
@@ -9,7 +9,7 @@
         "USE_SPHINX": "OFF",
         "CMAKE_BUILD_TYPE": "RelWithDebInfo",
         "CMAKE_C_FLAGS": "-Wall -Wextra -Wpedantic -Wshadow",
-        "CMAKE_CXX_FLAGS": "-Wall -Wextra -Wpedantic -Wshadow -D_GLIBCXX_ASSERTIONS",
+        "CMAKE_CXX_FLAGS": "-Wall -Wextra -Wpedantic -Wshadow -D_GLIBCXX_ASSERTIONS -fno-inline",
```

Comment:
What does this change do?

Reply:
it disable functions inlining to ease debugging